### PR TITLE
Common: fix infinite recursion

### PR DESCRIPTION
**Pull request by khancyr**
fix infinite recursion that show with newer sphinx versionx
```
writing output... [ 44%] docs/common-makeflyeasy-fighter-hand-throw
Extension error (sphinx.ext.mathjax):
Handler <function install_mathjax at 0x7f848975a840> for event 'html-page-context' threw an exception (exception: maximum recursion depth exceeded)
```

Recusion comes from the hidden toc on makefly rtf pages, there are two that are referencing each other.
Removing one, then trigger a not included into doctree issue, so I move up the hidden doctree to the common rtf page. This now trigger issue with missing include for blimp, thus the massive update on blimp

### CODE REVIEW

**Comment by Hwurzburg** [comment]
I dont understand where the recursion is without this PR that makes a lot of seemingly unrelated changes to add items to blimp

**Comment by khancyr** [comment]
Recusion comes from the hidden toc on makefly rtf pages, there are two that are referencing each other. 
Removing one, then trigger a not included into doctree issue, so I move up the hidden doctree to the common rtf page. This now trigger issue with missing include for blimp, thus the massive update on blimp

File: common/source/docs/common-rtf.rst [hunk 1]
```diff
@@ -64,9 +64,11 @@ VTOL/QuadPlanes from Partners
 * `FlyDragonDroneTech - FDG30 <https://www.droneassemble.com/product/vtol-uav-6-hours-endurance-with-1kg-payload-for-survey-serveillance/>`__
 * `FlyDragonDroneTech - FDG50F <https://www.droneassemble.com/product/hybrid-vtol-uav-7-hours-endurance-with-10kgs-payload/>`__
 * :ref:`Holybro Swan-K1 <common-Swan-K1>`
+* :ref:`MakeFLyEasy - Fighter Hand Throw <common-makeflyeasy-fighter-hand-throw>`
 * :ref:`MakeFLyEasy - Fighter VTOL <common-makeflyeasy-fighter-vtol>`
 * `MakeFLyEasy - Freeman 2300 <https://www.aliexpress.com/item/10000223137957.html?spm=a2g0o.store_home.productList_1076398524.pic_3>`__
 * `MakeFLyEasy - Freeman 2100 <https://www.aliexpress.com/item/10000223137957.html?spm=a2g0o.store_home.productList_1076398524.pic_2>`__
+* :ref:`MakeFLyEasy - Striver Mini Hand Throw <common-makeflyeasy-striver-mini-hand-throw>`
```
Comment (Hwurzburg) [inline common/source/docs/common-rtf.rst:71]: not a vtol, put under plane

File: common/source/docs/common-rtf.rst [hunk 2]
```diff
@@ -64,9 +64,11 @@ VTOL/QuadPlanes from Partners
 * `FlyDragonDroneTech - FDG30 <https://www.droneassemble.com/product/vtol-uav-6-hours-endurance-with-1kg-payload-for-survey-serveillance/>`__
 * `FlyDragonDroneTech - FDG50F <https://www.droneassemble.com/product/hybrid-vtol-uav-7-hours-endurance-with-10kgs-payload/>`__
 * :ref:`Holybro Swan-K1 <common-Swan-K1>`
+* :ref:`MakeFLyEasy - Fighter Hand Throw <common-makeflyeasy-fighter-hand-throw>`
```
Comment (Hwurzburg) [inline common/source/docs/common-rtf.rst:67]: not a vtol

File: common/source/docs/common-rtf.rst [hunk 3]
```diff
@@ -64,9 +64,11 @@ VTOL/QuadPlanes from Partners
...
 * :ref:`MakeFLyEasy - Fighter VTOL <common-makeflyeasy-fighter-vtol>`
 * `MakeFLyEasy - Freeman 2300 <https://www.aliexpress.com/item/10000223137957.html?spm=a2g0o.store_home.productList_1076398524.pic_3>`__
```
Comment (Hwurzburg) [inline common/source/docs/common-rtf.rst:69]: https://www.uavmodel.com/collections/vtol/products/makeflyeasy-freeman-4-1-2300mm-uav-vtol

not your issue but while we are doing this......

File: common/source/docs/common-rtf.rst [hunk 4]
```diff
@@ -64,9 +64,11 @@ VTOL/QuadPlanes from Partners
...
 * `MakeFLyEasy - Freeman 2100 <https://www.aliexpress.com/item/10000223137957.html?spm=a2g0o.store_home.productList_1076398524.pic_2>`__
```
Comment (Hwurzburg) [inline common/source/docs/common-rtf.rst:70]: https://www.uavmodel.com/products/makeflyeasy-striver-mini-4-1-4-2-2100mm-vtol-uav?variant=41656322654362

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

**Comment by Hwurzburg** [comment]
@khancyr still awaiting changes

**Comment by khancyr** [comment]
@Hwurzburg updated with corrections

File: common/source/docs/common-rtf.rst (hunk 4 above)
Comment (Hwurzburg) [reply common/source/docs/common-rtf.rst:71]: where is this model linked now?

**Review by Hwurzburg — APPROVED** [review]

**Pull request merged**